README.md

```diff
@@ -1,2 +1,17 @@
-# ng2-dashboard-cli
-yeoman generators for creating ng2 dashboards and components
+# Angular2 Dashboard Generator
+> yeoman generators for creating Angular2 dashboards and components
+
+## Usage
+Install `yo`,`angular-cli` and dashboard genertor `generator-ng2-dashboard-generator`
+```
+npm install -g yo angular-cli generator-ng2-dashboard-generator
+```
+
+Clone the github project https://github.com/akveo/ngx-admin , go to that project and run the following command
+
+Run `yo ng2-dashboard-generator`, optionally passing an app name:
+```
+yo ng2-dashboard-generator [module-name].json
```

Comment:
in my initial instructions, i mentioned that we are looking for a 'mocks' folder that will have one or many of these json files. so there's a number of ways you can do that, but one would be to write the logic to loop and read through the json files then wrap your existing logic in a function that you can call in that loop.